hypertrace-ingester/src/test/java/org/hypertrace/ingester/MigrationTest.java

```diff
@@ -83,542 +81,232 @@ public void resetSingleton()
         Map.of("schema.registry.url", "http://localhost:8081"));
   }
 
-  @Test
-  public void testHttpFields() throws Exception {
-    String tenantId = "tenant-" + random.nextLong();
+  @BeforeEach
+  public void setup()
+      throws SecurityException, NoSuchFieldException, IllegalArgumentException,
+          IllegalAccessException {
+    // Since JaegerToRawSpanConverter is a singleton, we need to reset it for unit tests to
+    // recreate the instance.
+    Field instance = JaegerSpanNormalizer.class.getDeclaredField("INSTANCE");
```

Comment:
unrelated to the change, since rather it's a code design issue. It's usually a testing problem with singletons. 

A singleton like `INSTANCE` usually hints towards a code design smell. The issue is described here perfectly https://stackoverflow.com/a/8263599/1439003